libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1455,10 +1471,15 @@ void IRGeneratorForStatements::declareAssign(IRVariable const& _lhs, IRVariable
 			_rhs.commaSeparatedList() <<
 			")\n";
 }
-void IRGeneratorForStatements::declare(IRVariable const& _var)
+
+IRVariable IRGeneratorForStatements::zeroValue(Type const& _type)
 {
-	if (_var.type().sizeOnStack() > 0)
-		m_code << "let " << _var.commaSeparatedList() << "\n";
+	IRVariable irVar{
+		"zero_value_for_type_" + _type.identifier() + m_context.newYulVariable(),
+		_type
+	};
+	m_code << "let " << irVar.commaSeparatedList() << " := " << m_utils.zeroValueFunction(_type) << "()\n";
```

Comment:
```suggestion
	define(irVar) << m_utils.zeroValueFunction(_type) << "()\n";
```
?
Still not sure if the IRVariable stuff is sufficiently intuitive and/or sufficiently documented yet :-).

Reply:
I think it just takes practice writing one or two PRs for the new cold gen to get used to it, I think it's fine